pkg/controller/middleware/auth.go

```diff
@@ -166,89 +169,31 @@ func (rah *RequireAuthHandler) Handle(next http.Handler) http.Handler {
 	})
 }
 
-type QuotaKeyIssuanceHandler struct {
-	ctx            context.Context
-	client         *auth.Client
-	db             *database.Database
-	quotaTTL       time.Duration // frequency of quota reset
-	requestsPerTTL int           // count of requests over the ttl window.
-	logger         *zap.SugaredLogger
+// UserRequestLimiter requires a user is authenticated using firebase auth,
```

Comment:
Do you plan on sharing this with other packages? If not, I'd recommend moving it inline into main.go.

Reply:
I moved this closer to where it is used. We can move it back up if needed.